### DISCUSSION TITLE
Recursively removing tag from objects present in bucket

### DISCUSSION THREAD

**Discussion by RitikRajSrivastava** [minio/minio#14034] [Q&A]
Having some 5000 objects in a bucket each object has a tag associated with it. Is there any way I can remove those tags recursively using the minio client ?

Thanks

**Comment by RitikRajSrivastava** [comment]
@harshavardhana can you please help !

**Reply by klauspost** [reply]
We will help as we have bandwidth. Please don't @ random devs.

No, there is currently no easy way to do that. `./mc find myminio/bucket/ --exec="./mc tag remove {}"` removes all tags, so you can use that if that is what you want. Tweak params as you need.

**Reply by RitikRajSrivastava** [reply]
Thanks @klauspost for the help :)
